### DISCUSSION TITLE
Suggestion for Improvement: Allow Use of the dub.sh Domain in Custom Projects

### DISCUSSION THREAD

**Discussion by deusdete** [dubinc/dub#379] [Ideas]
Hello everybody,

As an active user of this platform, I would like to share an observation I came across related to the use of domains in custom projects. Currently, I noticed a limitation that I believe could be improved to make the platform even more flexible.

In the free plan on the platform, shortened links are linked to the dub.sh domain, which is convenient for personal accounts. However, when creating custom projects, I noticed that the option to use the dub.sh domain is not available. Domain customization is critical to brand identity, but I believe the ability to choose between the dub.sh domain and a custom domain would also be valuable in custom projects.

A user can create a customized project to organize their work, using tags for example and have access to advanced resources, despite wanting to use the dub.sh domain.

I would like to suggest that the platform developer consider the possibility of allowing the use of the dub.sh domain in customized projects, offering greater flexibility to users and the ability to maintain a cohesive brand identity across all projects.

Thank you in advance for considering this suggestion. I hope this can be evaluated to further improve the experience of platform users.

I look forward to hearing the developer's response and other users' opinions on this issue.

**Comment by steven-tey** [comment]
Hey @deusdete! Thanks for the feature request. However, that's not possible currently – you can only set up custom project if you have a [custom domain](https://dub.sh/domains) – you can learn more about custom projects on Dub as well as its benefits here: https://dub.co/help/article/what-is-a-project

That being said, this is not the first time we've heard this so would love to learn more about the value of using `dub.sh` links in a team/custom project. Isn't it much better to have custom branded links than the default `dub.sh` links? 👀

**Reply by deusdete** [reply]
Hi @steven-tey! I imagine that there may be users/clients who do not have domains to configure or even do not want to use a custom domain. They just want to have the features that a Pro project brings us, examples:

1 - Remove the QR logo: this is a feature that would only require one project and the Pro version.
2 - Tags: we need the project to link these tags
3 - Geographic segmentation: user would also need to have a project with the Pro version.
4 - Link-in-bio pages (future resources): like Linktree, the user creates their page and can access it through the URL dub.sh/deusdete.

All of these features are incredibly interesting with or without the custom domain.

But, I understand that currently (in the code) there are several parts that depend on the "domain" field to search for something in the database relating to that project.
